### ISSUE TITLE
PAR Dominant inheritance

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7490]
**Preferred term label:**
PAR Dominant inheritance

**Synonyms**
Pseudo Autosomal Region Dominant inheritance

**Definition (free text, please give PubMed ID)**
 Gonosomal inheritance that is dominant and in which heterozygous males and females both manifest a disease phenotype

**Parent term (use hpo.jax.org/app)**
Gonosomal inheritance HP:0010985

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034340
